johnzon-osgi/src/main/java/org/apache/johnzon/osgi/Activator.java

```diff
@@ -90,6 +133,20 @@ public void start(BundleContext context) throws Exception {
                     )
                 )
             ),
+            ignore(
+                highest(serviceReferences(ClientBuilder.class)).flatMap(OSGi::prototypes).map(
+                    serviceObjects -> new JsonClientBuilderFactory(context, serviceObjects)
+                ).flatMap(
+                    factory -> register(
```

Comment:
Looks wrong in a json layer to register a jaxrs component, should stay in jaxrs whiteboard which should pull providers as per jaxrs spec imho, wdyt?

Reply:
hey @rmannibucau I sent the wrong branch in this PR. let's not get ahead of ourselves just yet. :) I will close this and send it in the order I intended.